src/pages/TheaterResources.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import PageContainer from '../components/layout/PageContainer';
+import { Tagline, Title } from '../components/layout/Titles';
+import SVGLayer from '../components/SVGLayer';
+import blueBlob from '../images/blue_blob.svg';
+import streamingDance from '../images/streaming_dance.svg';
+
+const TheaterResources = () => {
+  return (
+    <PageContainer>
+      <Row>
+        <Col lg={8}>
+          <Title>THEATER RESOURCES</Title>
+          <Tagline>Submit or view a resource!</Tagline>
+          <p className="margin-container">
+            Our collection of useful links and resources for actors, directors,
+            producers and crew. Do you have a resource you think belongs on this
+            list? Add it to our list by clicking
+            <a href="https://forms.gle/eCHjeDGphFBr7y4W6"> here </a>
+            or filling out the form below.
+          </p>
+          <table className="table">
+            <thead>
+              <tr>
+                <th scope="col">Organization</th>
+                <th scope="col">Description</th>
+                <th scope="col">Category</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <a href="https://www.techsoup.org/"> Tech Soup (Mock)</a>
+                </td>
+                <td>
+                  Discounts on software and technology for nonprofits from
+                  leading brands
+                </td>
+                <td>Technology</td>
+              </tr>
+              <tr></tr>
+              <tr></tr>
+            </tbody>
+          </table>
+          <br></br>
```

Comment:
One note before we merge, actually: let's avoid using br's as spacing and use CSS. You can add classes or style-components to the table to give it a bottom margin